### ISSUE TITLE
Can not view application

### ISSUE THREAD

**Issue by stealthrabbi** [myliang/fish-admin#2]
I follow the instructions, starting the back end with bootRun, and using npm to install and start the dev server. I get directed to http://localhost:8082/, and the browser displays Cannot GET /

**Comment by myliang** [comment]
execute init.sql

**Comment by stealthrabbi** [comment]
Execute it where? Can the instruction specify this? What DB engine does this use?

**Comment by myliang** [comment]
https://github.com/myliang/fish-admin/tree/master/api/db

**Comment by stealthrabbi** [comment]
Yes, I see where the file is. However, it's not clear to me where to execute it, and it's not mentioned in your Readme's quickstart guide. See previous comment.

**Comment by myliang** [comment]
db: postgresql
```sql
create database fish;
```
https://github.com/myliang/fish-admin/blob/master/api/src/main/resources/application.properties
